### PR TITLE
Remove -precomposed rel

### LINKED ISSUE #26707: link: fix examples for Apple touch icon
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link

### What specific section or headline is this issue about?

Providing icons for different usage contexts

### What information was incorrect, unhelpful, or incomplete?

Apple devices no longer check for `apple-touch-icon-precompose`

> In older versions of Safari for iOS, the [precomposed keyword](https://mathiasbynens.be/notes/touch-icons#effects) could be used to prevent iOS from adding different visual effects to the touch icon (i.e., rounded corners, drop shadow, reflective shine). Starting with iOS 7, no special effects are applied to touch icons, so there is no need to use the precomposed keyword anymore.

https://github.com/webhintio/hint/blob/main/packages/hint-apple-touch-icons/README.md#why-is-this-important

### What did you expect to see?

```
<link
  rel="apple-touch-icon"
  sizes="144x144"
  href="favicon144.png" />
...

```

### Do you have any supporting links, references, or citations?

https://github.com/webhintio/hint/blob/main/packages/hint-apple-touch-icons/README.md#why-is-this-important

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/link`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/link/index.md
* Last commit: https://github.com/mdn/content/commit/fa8a44b8bff24a4032181c4fd155c459c0dc9161
* Document last modified: 2023-03-13T07:09:38.000Z

</details>

### PR DESCRIPTION
**Pull request by CBID2**
### Description
This is a continuation of a task I started in another PR. See the Description section in PR #27935 

### Motivation
The keyword is no longer needed according to MDN’s accessibility expert. To get more context, check out the source I linked in this section on PR #27935 

### Related issues and pull requests

 Relates to #27935
Fixes #26707

### CODE REVIEW

File: files/en-us/web/html/element/link/index.md [hunk 1]
```diff
@@ -323,19 +323,13 @@ You can include links to several icons on the same page, and the browser will ch
...
-  rel="apple-touch-icon-precomposed"
-  sizes="114x114"
-  href="favicon114.png" />
+<link rel="apple-touch-icon" sizes="114x114" href="favicon114.png" />
 <!-- first- and second-generation iPad: -->
-<link rel="apple-touch-icon-precomposed" sizes="72x72" href="favicon72.png" />
+<link rel="apple-touch-icon" sizes="72x72" href="favicon72.png" />
 <!-- non-Retina iPhone, iPod Touch, and Android 2.1+ devices: -->
-<link rel="apple-touch-icon-precomposed" href="favicon57.png" />
+<link rel="apple-touch" href="favicon57.png" />
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/link/index.md:332]:
```suggestion
<link rel="apple-touch-icon" href="favicon57.png" />
```

**Review by Josh-Cena — APPROVED** [review]
LGTM thanks

**Comment by CBID2** [comment]
> LGTM thanks

Thanks @Josh-Cena. Should I delete the branch in the closed PR?

**Comment by Josh-Cena** [comment]
You are free to; it wouldn't affect us.

**Pull request merged**